localisation/00_zuwc_united_localisation.csv

```diff
@@ -3058,23 +3058,23 @@ DYN12_DESC;With a score of $SCORE$ you$FAILED$ beat House Mechagon. In the dista
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+GetClassSkill_meditation_and_martial_arts;meditation and martial arts;;;;;;;;;;;;;x
+A_MONK;A monk.;;;;;;;;;;;;;x
+teach_yourself_monk;Train Monk Skills on your own;;;;;;;;;;;;;x
+teach_yourself_monk_desc;Train monk skills on your own and become a monk.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1600;You decide to start with meditation practice. You've heard that the most important thing is to control your breathing. You sit in the lotus position and begin to breathe in and out evenly. Your training will take some time indeed.;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1700;You've finally mastered the very last thing: the martial arts of bare-handed combat. Your hands hurt, but you feel stronger. Of course, you understand your training isn't over. The path to excellence may take your entire life, but you are happy with today's result.;;;;;;;;;;;;;x
+training_self_monk;Training Monk Skills;;;;;;;;;;;;;x
+training_mentor_monk;Training Monk Skills;;;;;;;;;;;;;x
+is_monk_class_trigger_tooltip;Is monk;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1605;[Root.PrimaryTitle.GetFOA], I would be glad to heal your stress and depression with the help of my magic if you let me.\n\nSigned,\n[From.GetTitledName];;;;;;;;;;;;;x
+soothing_mist;Wave Soothing Mist;;;;;;;;;;;;;x
+soothing_mist_desc;Wave soothing mist healing the stress and depression of �Y[Root.GetBestName]�!.;;;;;;;;;;;;;x
+text_effect_soothing_mist;Enables �GWave Soothing Mist�! on character portraits;;;;;;;;;;;;;x
+EVTDESC_WCCLS_1610;[target_class.GetFirstName] fights in the heat of battle alongside [target_class.GetHerHis] soldiers. After the first unfortunate enemy soldier dies by [target_class.GetHerHis] hand, [target_class.GetSheHe] calls three elemental spirits to aid in the battle. They immediately rush into combat, knocking the enemy back.;;;;;;;;;;;;;x
```

Comment:
Is this how it worked in WoW? Because in WC3, he split himself into the 3 elementals...

Reply:
Arithon is right. 

> Split into 3 elemental spirits for 15 sec, each spirit dealing 45% of normal damage and healing. You directly control the Storm spirit, while Earth and Fire spirits mimic your attacks on nearby enemies.

Reply:
The event together with localization is reworked in cc27490f6d5b72ba63e5736325b44cf3ae3edc73.